ddtrace/internal/utils/time.py

```diff
@@ -1,20 +1,41 @@
 from datetime import datetime
+from datetime import timedelta
 from types import TracebackType
 from typing import Optional
 from typing import Type
 
 from ddtrace.internal import compat
+from ddtrace.internal.logger import get_logger
+
+
+log = get_logger(__name__)
+
+
+def fromisoformat_py2(t):
+    # type: (str) -> datetime
+    ret = datetime.strptime(t[0:19], "%Y-%m-%dT%H:%M:%S")
```

Comment:
can you include some example strings in either doc string or comments here of in `parse_isoformat`. it'll make the code easier to read if you know the type of string it is parsing.